pkg/fleet/installer/service/datadog_agent.go

```diff
@@ -95,7 +96,7 @@ func SetupAgent(ctx context.Context) (err error) {
 			return
 		}
 	}
-	if err = createAgentSymlink(ctx); err != nil {
+	if err = os.Symlink("/opt/datadog-packages/datadog-agent/stable/bin/agent/agent", agentSymlink); err != nil {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 Warning:** ‏this isn't a force symlink by default (= if symlink already exists it errors), we can `ln -sf` instead